recipes/expertai-extract/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License

```

Comment:
Could you please switch to the GitHub source release so that the License file is extracted from the source instead of included here in the recipe? That way, if the project license changes in the future, it will be automatically updated for the conda-forge release too.

Reply:
Hi, Daniel. Could you please, clarify this one for us? Our GitHub project source is here: https://github.com/therealexpertai/extract-python. And the LICENSE file is there originally. We have manually copied it and put it here in the recipe. What actions exactly do  we need to conduct? Is removing the LICENCE file from here enough or we should do something extra in order to fulfill your requirements?

Reply:
@carterbox Hi Daniel, Could you please give us a reply on this and the other question of mine? We would like to advance on this merge request and I don't know whether something else is needed from my side?

Reply:
Your PYPI source does not include the file named "LICENSE", so you needed to include the "LICENSE" file in this PR. If you get the source from GitHub instead, using the URL like this: https://github.com/therealexpertai/extract-python/archive/refs/tags/0.1.1.tar.gz, then the "LICENSE" file is included in the archive, so you don't have to manually include it here.